src/logging.js

```diff
@@ -34,45 +34,44 @@ function getFormatters() {
     return winston.format.combine(...formatters);
 }
 
-function getTransports(isDev: boolean): Array<Transport> {
+function getTransport(isDev: boolean): Transport {
     if (process.env.NODE_ENV === "test") {
         // During testing, we just dump logging to a stream.
         // This isn't used for anything at all right now, but we could use
         // it for snapshot testing with some updates at some point.
         const sink = new stream.Writable({write: () => {}});
         // This is a hack to make our writable stream work $FlowFixMe
         sink._write = sink.write;
-        return [
-            new winston.transports.Stream({
-                format: getFormatters(),
-                stream: sink,
-            }),
-        ];
+        return new winston.transports.Stream({
+            format: getFormatters(),
+            stream: sink,
+        });
     }
 
     /**
      * If we're in dev mode, just use a console transport.
      */
     if (isDev) {
-        return [
-            new winston.transports.Console({
-                format: getFormatters(),
-            }),
-        ];
+        return new winston.transports.Console({
+            format: getFormatters(),
+        });
     }
 
     /**
-     * We must be in production, so use the Stackdriver logging setup.
+     * We must be in production, so we will use the Stackdriver logging setup.
+     *
+     * To avoid getting duplicate entries, make sure this transport is passed
+     * to the middleware method below.
      */
-    return [new lw.LoggingWinston()];
+    return new lw.LoggingWinston();
 }
 
 function initLogging(logLevel: LogLevel, isDev: boolean): Logger {
     // This is the logger that we use to log general information in our app.
     // Whereever one might use console, use this instead.
     const winstonLogger = winston.createLogger<NpmLogLevels>({
         level: logLevel,
-        transports: getTransports(isDev),
+        transports: getTransport(isDev),
```

Comment:
Why not just say `transports: [getTransport(isDev)]` and keep the type signature the same as before?

Reply:
Oh, so the logger API actually accepts either (I had just been lazy with the flow types earlier). Seemed silly to create an array when we didn't need one.